exonum_client/client.py

```diff
@@ -311,6 +318,31 @@ def get_instance_id_by_name(self, name: str) -> Optional[int]:
 
         return None
 
+    def send_websocket_transaction(self, message: ExonumMessage) -> str:
```

Comment:
Hm. What's the point of this being a separate method with separate connection, if we can send it via HTTP? I think it should be a `Subscriber` method.

Reply:
OK, I'll move it to `Subscriber` class.

Reply:
Done.